README.md

```diff
@@ -50,5 +50,9 @@ The supported cloud providers and their respective metadata are as follows:
       - COREOS_OPENSTACK_IPV4_LOCAL
       - COREOS_OPENSTACK_IPV4_PUBLIC
       - COREOS_OPENSTACK_INSTANCE_ID
+  - virtualbox
+    - Attributes
+      - COREOS_VIRTUALBOX_HOSTNAME
+      - COREOS_VIRTUALBOX_IPV4_PRIVATE (only via vagrant)
```

Comment:
> only via vagrant

This alone seems to indicate that the platform should incorporate both virtualbox and vagrant.